orangecontrib/text/widgets/owcorpusviewer.py

```diff
@@ -115,7 +115,8 @@ def load_features(self):
         self.selected_features = []
         if self.corpus is not None:
             self.features = [(meta.name, vartype(meta)) for meta in self.corpus.domain.metas] + \
-                            [(attr.name, vartype(attr)) for attr in self.corpus.domain.attributes]
+                            [(attr.name, vartype(attr)) for attr in self.corpus.domain.attributes] + \
+                            [(class_var.name, vartype(class_var)) for class_var in self.corpus.domain.class_vars]
```

Comment:
There's `Domain.variables` which `== Domain.attributes + Domain.class_vars`. I wouldn't say one is preferred to the other, though, or more consistently used.

If this is fine with you, it sure is not with @VesnaT. Them lines far too long. :smiling_imp: